extensions/data-plane/data-plane-http/src/main/java/org/eclipse/edc/connector/dataplane/http/DataPlaneHttpExtension.java

```diff
@@ -32,16 +32,18 @@
 import org.eclipse.edc.spi.system.ServiceExtensionContext;
 import org.eclipse.edc.spi.types.TypeManager;
 
+import static java.lang.Integer.parseInt;
+
 /**
  * Provides support for reading data from an HTTP endpoint and sending data to an HTTP endpoint.
  */
 @Provides(HttpRequestParamsProvider.class)
 @Extension(value = DataPlaneHttpExtension.NAME)
 public class DataPlaneHttpExtension implements ServiceExtension {
     public static final String NAME = "Data Plane HTTP";
-    private static final int DEFAULT_PART_SIZE = 5;
+    private static final String DEFAULT_PARTITION_SIZE = "5";
```

Comment:
Please don't chnage the type and parse it below. The annotation type system is more restrictive, so create another constant that converts the typed int value to a string.

Reply:
you can do `DEFAULT_PART_SIZE + ""` to convert to string

Reply:
Done. Should I also go and fix similar doc/config/parse stuff in `CoreDefaultServicesExtension` and other places throughout the project, to make it uniform? Or rather new issue(s) should be created, to avoid broad-scoped PR?